### PR TITLE
Modified exercise search routine to return only the latest exercises

### PR DESCRIPTION
**Pull request by Dantemss**
Ready for review.

### CODE REVIEW

File: app/routines/search_exercises.rb [hunk 1]
```diff
@@ -15,28 +15,18 @@ class SearchExercises
...
-                                                   copyright_holders: :user,
-                                                   editors: :user],
-                                     questions: [
-                                       :hints,
-                                       answers: :stem_answers,
-                                       stems: [:stylings, :combo_choices]
-                                     ]
-                                   ),
+
+    latest_only = true
```
Comment (jpslav) [inline app/routines/search_exercises.rb]: Took me a sec to figure out what the `latest_only`s were doing.  Can you add a comment here along the lines of:

```ruby
# By default, only return the latest exercises.  If certain search conditions are specified,
# this "latest" condition may be disabled.
```

File: lib/active_record/calculations_patch.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+ActiveRecord::Calculations.class_exec do
```
Comment (jpslav) [inline lib/active_record/calculations_patch.rb]: ＼　　　　　　☆
　　　　　　　　　　　　　|　　　　　☆
　　　　　　　　　　(⌒ ⌒ヽ　　　/
　　　　＼　　（´⌒　　⌒　　⌒ヾ　　　／
　　　　　 （’⌒　;　⌒　　　::⌒　　）
　　　　　（´　　　　　）　　　　　:::　）　／
　　☆─　（´⌒;:　　　　::⌒`）　:;　　）
　　　　　（⌒::　　　::　　　　　::⌒　）
　　 　／　（　　　　ゝ　　ヾ　丶　　ソ　─

File: app/routines/search_exercises.rb (hunk 1 above)
Comment (Dantemss) [inline app/routines/search_exercises.rb]: OK

**Comment by jpslav** [comment]
Looks good.  You are the man.

**Pull request merged**